source/retryable-writes/tests/README.rst

```diff
@@ -74,7 +74,8 @@ failCommand
 Some tests depend on a server fail point, ``failCommand``, which allows the
 client to force the server to return an error. Unlike
 ``onPrimaryTransactionalWrite``, ``failCommand`` does not allow the client to
-control whether the server will commit the operation. See:
+directly control whether the server will commit the operation (execution of the
+write depends entirely on whether the ``errorCode`` option is specified). See:
```

Comment:
Execution of the write also depends on whether the "closeConnection" option is specified. If "closeConnection" is specified, the write will not be executed.

Reply:
Thanks for confirming. Changed `closeConnection` docs to:

>  If ``true``, the command will not be executed, the connection will be closed, and the client will see a network error.

...and updated this to state that it depends on `closeConnection` and/or `errorCode` being set.